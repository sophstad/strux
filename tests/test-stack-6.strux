int main() {
	Stack::string s = new Stack::string("hello", "world");
	string str = s.peek();
	print(str); 

	Stack::num s2 = new Stack::num(3.0, 4.0);
	num n = s2.peek();
	print(n); 

	Stack::int s3 = new Stack::int(3, 4, 5);
	int i = s3.peek();
	print(i); 

	Stack::bool s4 = new Stack::bool(true, false);
	bool b = s4.peek();
	print(b);
	return 0;
}
