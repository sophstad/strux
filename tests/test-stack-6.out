world
4.000000
5
0
